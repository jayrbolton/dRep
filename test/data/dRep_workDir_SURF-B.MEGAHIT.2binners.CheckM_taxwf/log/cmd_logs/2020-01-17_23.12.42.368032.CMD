/miniconda/bin/checkm taxonomy_wf domain Bacteria /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/prodigal/ /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir/ -f /kb/module/work/tmp/dRep_workDir_d7f80abb-1c52-43db-9fae-de726bb861a2/data/checkM/checkM_outdir//results.tsv --tab_table -t 8 -g -x faa
