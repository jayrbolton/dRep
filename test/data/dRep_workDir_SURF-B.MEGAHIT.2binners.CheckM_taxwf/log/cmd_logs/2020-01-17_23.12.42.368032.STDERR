    Finished processing 0 of 11 (0.00%) bins.    Finished processing 1 of 11 (9.09%) bins.    Finished processing 2 of 11 (18.18%) bins.    Finished processing 3 of 11 (27.27%) bins.    Finished processing 4 of 11 (36.36%) bins.    Finished processing 5 of 11 (45.45%) bins.    Finished processing 6 of 11 (54.55%) bins.    Finished processing 7 of 11 (63.64%) bins.    Finished processing 8 of 11 (72.73%) bins.    Finished processing 9 of 11 (81.82%) bins.    Finished processing 10 of 11 (90.91%) bins.    Finished processing 11 of 11 (100.00%) bins.
    Finished parsing hits for 1 of 11 (9.09%) bins.    Finished parsing hits for 2 of 11 (18.18%) bins.    Finished parsing hits for 3 of 11 (27.27%) bins.    Finished parsing hits for 4 of 11 (36.36%) bins.    Finished parsing hits for 5 of 11 (45.45%) bins.    Finished parsing hits for 6 of 11 (54.55%) bins.    Finished parsing hits for 7 of 11 (63.64%) bins.    Finished parsing hits for 8 of 11 (72.73%) bins.    Finished parsing hits for 9 of 11 (81.82%) bins.    Finished parsing hits for 10 of 11 (90.91%) bins.    Finished parsing hits for 11 of 11 (100.00%) bins.
    Finished processing 0 of 11 (0.00%) bins.    Finished processing 1 of 11 (9.09%) bins.    Finished processing 2 of 11 (18.18%) bins.    Finished processing 3 of 11 (27.27%) bins.    Finished processing 4 of 11 (36.36%) bins.    Finished processing 5 of 11 (45.45%) bins.    Finished processing 6 of 11 (54.55%) bins.    Finished processing 7 of 11 (63.64%) bins.    Finished processing 8 of 11 (72.73%) bins.    Finished processing 9 of 11 (81.82%) bins.    Finished processing 10 of 11 (90.91%) bins.    Finished processing 11 of 11 (100.00%) bins.
    Finished processing 0 of 11 (0.00%) bins.    Finished processing 1 of 11 (9.09%) bins.    Finished processing 2 of 11 (18.18%) bins.    Finished processing 3 of 11 (27.27%) bins.    Finished processing 4 of 11 (36.36%) bins.    Finished processing 5 of 11 (45.45%) bins.    Finished processing 6 of 11 (54.55%) bins.    Finished processing 7 of 11 (63.64%) bins.    Finished processing 8 of 11 (72.73%) bins.    Finished processing 9 of 11 (81.82%) bins.    Finished processing 10 of 11 (90.91%) bins.    Finished processing 11 of 11 (100.00%) bins.
    Finished parsing hits for 1 of 11 (9.09%) bins.    Finished parsing hits for 2 of 11 (18.18%) bins.    Finished parsing hits for 3 of 11 (27.27%) bins.    Finished parsing hits for 4 of 11 (36.36%) bins.    Finished parsing hits for 5 of 11 (45.45%) bins.    Finished parsing hits for 6 of 11 (54.55%) bins.    Finished parsing hits for 7 of 11 (63.64%) bins.    Finished parsing hits for 8 of 11 (72.73%) bins.    Finished parsing hits for 9 of 11 (81.82%) bins.    Finished parsing hits for 10 of 11 (90.91%) bins.    Finished parsing hits for 11 of 11 (100.00%) bins.
